extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -81,6 +110,9 @@ export class ValidationBubble {
  * @private
  */
 function hideBubble(state) {
+  state.bubbleElement.removeAttribute('aria-alert');
+  state.bubbleElement.removeAttribute('role');
+  removeChildren(state.bubbleElement);
```

Comment:
isn't just `aria-hidden` enough? (needs `aria-hidden` regardless of whether you still need to remove the role or not)